templates/home.html

```diff
@@ -0,0 +1,136 @@
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+<script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Bootstrap Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+
+<style type="text/css">
+
+body{
+  background-image: url(https://truecommerce.ecutopia.com/wp-content/uploads/2019/02/ecUtopia-Web-Vendor-Portal-Background.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+</style>
+
+</head>
+<body>
+
+
+<div class="container" style="margin-top: 5%;">
+  <div class="row">
+    <div class="col-sm-4"> </div>
+<div class="col-md-4">
+
+<h1 class="text-center text-success"> Login page</h1>
+<br/>
+
+<div class="col-sm-12">
+
+  <ul class="nav nav-pills" >
+
```

Comment:
Why do you have too much line spacing here?

Reply:
easier to read the code like that

Reply:
you should decide on coding style
usually extra newlines are discouraged

Reply:
@yuvalk You want me to change it?